susemanager-utils/susemanager-sls/src/_modules/uyuni_users.py

```diff
@@ -0,0 +1,825 @@
+
+            :param minion_ids:
+            :return:
+            """
+            minions = {}
+            for mid in minion_ids:
+                minions[mid] = self._get_key_fingerprint(minion_id=mid)
+
+            return minions
+
+    def __init__(self, pillar: Optional[Dict[str, Any]] = None):
+        UyuniRemoteObject.__init__(self, pillar=pillar)
+        self._minions = UyuniChildMasterIntegration.FCkMinions(salt.config.master_config(self._get_master_config()))
+
+    @staticmethod
+    def _get_master_config() -> str:
+        """
+        Return master config.
+        :return:
+        """
+        cfg_path = UyuniChildMasterIntegration.DEFAULT_MASTER_CONFIG_PATH
+        for path in __pillar__.get("uyuni", {}).get("masters", {}).get("configs", [cfg_path]):
+            if os.path.exists(path):
+                cfg_path = path
+                break
+
+        return cfg_path
+
+    def list_minions(self, active: bool = False) -> List[str]:
+        """
+        Return list of currently registered minions.
+
+        :param active: Return only active minions.
+        :return: list of minion ids
+        """
+        return self._minions.connected_ids() if active else self._minions._pki_minions()
+
+    def list_minions_fp(self, active: bool = False) -> Dict[str, str]:
+        """
+        Return list of currently registered minions, including their key fingerprints.
+
+        :param active: Return only active minions.
+        :return: mapping of minion ids to the fingerprints
+        """
+        return self._minions._get_fingerprints(self.list_minions(active=active))
+
+    def select_minions(self, expr: str, tgt: str = "glob") -> Dict[str, Union[List[str], bool]]:
+        """
+        Select minion IDs that matches the expression.
+
+        :param expr: expression
+        :param tgt: target type, one of the following: glob, grain, grain_pcre, pillar, pillar_pcre,
+                    pillar_exact, compound, compound_pillar_exact. Default: glob.
+
+        :return: list of minions
+        """
+        return self._minions.check_minions(expr=expr, tgt_type=tgt)
+
+    def select_minions_fp(self, expr: str, tgt: str = "glob") -> Dict[str, Union[str, bool]]:
+        """
+        Select minion IDs that matches the expression.
+
+        :param expr: expression
+        :param tgt: target type, one of the following: glob, grain, grain_pcre, pillar, pillar_pcre,
+                    pillar_exact, compound, compound_pillar_exact. Default: glob.
+
+        :return: mapping of minion ids to the fingerprints
+        """
+        selected = self.select_minions(expr=expr, tgt=tgt)
+        ret = {
+            "minions": self._minions._get_fingerprints(selected["minions"]),
+            "missing": self._minions._get_fingerprints(selected["missing"]),
+            "ssh_minions": selected.get("ssh_minions", False)
+        }
+
+        return ret
+
+    def master(self, *args: List[str], **kwargs: Dict[str, Any]):
```

Comment:
What is the expected use case for this method?

Reply:
I demoed it the other day.

Reply:
Do I get it correctly that this is not really related to the main purpose of this PR, namely management of groups, users and orgs?

Reply:
How it is not related? We were discussing that for long [here](https://github.com/SUSE/spacewalk/issues/9876). Sure, if you think this is no longer needed, feel free to remove it. :man_shrugging: